Code/GraphMol/Kekulize.cpp

```diff
@@ -347,6 +348,13 @@ bool kekulizeWorker(RWMol &mol, const INT_VECT &allAtms,
           // mol.debugMol(std::cerr);
           numBT++;
         } else {
+          // undo any remaining changes we made while here
```

Comment:
Yeah, it's when the addition of double bonds has failed, but no choices have been made that can be backtracked from.